deploy/osps/default/osp-rhel.yaml

```diff
@@ -93,8 +93,17 @@ spec:
               yum install -y curl jq
 
               curl -s -k -v --header 'Authorization: Bearer {{ .Token }}' {{ .ServerURL }}/api/v1/namespaces/cloud-init-settings/secrets/{{ .SecretName }} | jq '.data["cloud-config"]' -r| base64 -d > /etc/cloud/cloud.cfg.d/{{ .SecretName }}.cfg
+
               cloud-init clean
+              {{- if (eq .CloudProviderName "azure") }}
+              sudo systemctl stop NetworkManager
+              sudo ip addr flush dev eth0 && sudo ip route flush dev eth0
```

Comment:
why is this needed? and notice that this is not persisted, i.e. that will not work after the reboot.